tests/test_radiosity.cpp

```diff
@@ -94,9 +95,9 @@ TEST_CASE("Form factor of two orthogonal unit squares", "[form_factor]") {
     float F_ij, F_ji;
     std::tie(F_ij, F_ji) = orthogonal_scenario(1, 1, 1);
 
-    constexpr float F_EXPECTED = 0.2;
-    REQUIRE(F_ij == Approx(F_EXPECTED).epsilon(TOLERANCE));
-    REQUIRE(F_ji == Approx(F_EXPECTED).epsilon(TOLERANCE));
+    constexpr float F_EXPECTED = 0.20004f;
```

Comment:
It's page 100. Also, for parallel they write 0.200043 on page 96 in figure 4.22. but that's wrong.